.changeset/cool-mice-sit.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-tech-insights': minor
```

Comment:
This package is at `0.x` and there's nothing breaking right? In that case this should be `patch`.
For a package on `1.x` and above it should indeed be minor.